packages/components-native/src/TextList/__snapshots__/TextList.test.tsx.snap

```diff
@@ -16,13 +16,15 @@ exports[`TextList when the bulletItems is provided displays the text list 1`] =
         {
           "padding": 0,
         },
+        undefined,
```

Comment:
`Content` is in `TextList` & even though `UNSAFE_` isn't being used, `View` is still including the prop in the style array as `undefined`. No side effect in rendering, etc.